docs/reference-guides/data/data-core.md

```diff
@@ -192,11 +192,11 @@ Returns the loaded entities for the given kind.
 _Parameters_
 
 -   _state_ `State`: Data state.
--   _kind_ `Kind`: Entity kind.
+-   _kind_ `ET.Kind`: Entity kind.
```

Comment:
It's not ideal that the docs mention both ET.Kind and Kind, but tradeoffs, tradeoffs 🤷